tests/optimizer/test_scipy.py

```diff
@@ -0,0 +1,99 @@
+from typing import Callable, Optional
+
+import pytest
+
+from tensorwaves.interfaces import Estimator
+
+try:
```

Comment:
Is this try-catch block needed in the tests? I would say to run the tests, one needs to have the developer or `requirements-test.txt`  dependencies installed anyway, because the tests are not installed.

Reply:
The thing is the test discovery will fail if this try except block is not there

Reply:
But the only circumstance under which you want to discover tests is if you have `pytest` installed. And I think that should be done through installing `reqs/3.x/requirements-test.txt`.

Reply:
Thats true. Let me fix this then

Reply:
d41e7fc44dd46024c5684f859eac820578438848